app/views/navigation_rules/edit.html.erb

```diff
@@ -18,7 +18,7 @@
 <% content_for :title, @step_by_step_page.title %>
 <% content_for :context, "Sidebar settings" %>
 
-<%= render "shared/steps/form_notice", message: notice, status: "success" %>
+<%= render "shared/steps/form_notice", message: notice, status: nil %>
```

Comment:
Is this an intentional change?

Reply:
no, one of the rake task was force causing this. 
I have undo this change. good spot